integreat_compass/cms/urls/protected.py

```diff
@@ -25,5 +25,27 @@
                 ),
             ]
         ),
-    )
+    ),
+    path(
+        "applications/",
+        include(
+            [
+                path(
+                    "votes/",
+                    include(
+                        [
+                            path("", interactions.VoteFormView.as_view(), name="votes"),
+                            path(
+                                "<int:offer_version_id>/add_vote/",
+                                interactions.VoteFormView.add_vote,
+                                name="add_vote",
+                            ),
+                        ]
+                    ),
+                ),
+                path("reports/", interactions.VoteFormView.as_view(), name="reports"),
+                path("declined/", interactions.VoteFormView.as_view(), name="declined"),
+            ]
+        ),
+    ),
```

Comment:
Thanks, I think this makes a lot more sense. We could even move the `/applications/votes` view to just `/applications` (since there's nothing there). (And then `applications/votes/add_vote` to `applications/add_vote`.)

Just like the other comment though, not strictly necessary, this would work too.